String/string.cpp

```diff
@@ -57,16 +57,16 @@ char String::charAt(int index) {
 }
 
 int String::compareTo(String anotherString){
+	if(size > anotherString.size){
+		return size-anotherString.size;
+	}
+	else if(size < anotherString.size){
+		return size-anotherString.size;
+	}
```

Comment:
Sizes are unchanged throughout the whole function body. No need to compare them every iteration of the for-loop.
Compare it once, outside the for-loop.